### ISSUE TITLE
Increase contrast of theme colors for improved accessibility

### ISSUE THREAD

**Issue by rc-ms** [microsoft/electionguard#65]
## Bug Report

**Current Behavior**
Existing nav and theme formatting have insufficient contrast for accessibility; need to remove copyright info

**Expected behavior/code**
UX should exhibit contrast ratio of 4.5:1 for all text foreground/background display

**Environment**
- OS: [e.g. OSX 10.13.4, Windows 10]
- Version: [e.g. 1.0.1]

**Possible Solution**
increase contrast ratio by changing header and default
<img width="1358" alt="Screen Shot 2021-02-17 at 2 19 57 PM" src="https://user-images.githubusercontent.com/33664020/108275544-5c3d8480-712b-11eb-8a83-774474359686.png">

 teal colors

**Additional context/Screenshots**
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by keithrfung** [comment]
This will likely mean customizing the colors completely as we currently use the built in teal. 
https://github.com/microsoft/electionguard/blob/807d9d10bb3c82f4abfe3aff2859b8fecc80dd2a/mkdocs.yml#L50

We will likely have to built up a custom color scheme to handle this issue since the tool is catching all the built in's. 
https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#custom-color-schemes

```
[data-md-color-scheme="electionguard"] {
  --md-primary-fg-color:        #INSERT_COLOR;
  --md-primary-fg-color--light: #INSERT_COLOR;
  --md-primary-fg-color--dark:  #INSERT_COLOR;
}
```
